src/linkgraph/linkgraph_gui.cpp

```diff
@@ -223,7 +223,7 @@ void LinkGraphOverlay::AddLinks(const Station *from, const Station *to)
 		if (lg[ge.node].HasEdgeTo(to->goods[c].node)) {
 			ConstEdge &edge = lg[ge.node][to->goods[c].node];
 			this->AddStats(c, lg.Monthly(edge.capacity), lg.Monthly(edge.usage),
-					ge.flows.GetFlowVia(to->index),
+					ge.HasData() ? ge.GetData().flows.GetFlowVia(to->index) : 0,
```

Comment:
No idea whether it has any merit, but with the `std::ranges::find` PR you were talking about projections. So, might that be something to consider here?
I think `ge.HasData() ? ge.GetData().x : y` might be more legible when written as `ge.GetDataOrElse(&x, y)`. I haven't checked whether that'll work in this instance, or whether it's worth abstracting this.